assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -11,6 +11,23 @@
 	--masterbar-toggle-drafts-editor-background: darken( $blue-wordpress, 12% );
 	--masterbar-toggle-drafts-editor-border-color: darken( $blue-wordpress, 5% );
 	--masterbar-toggle-drafts-editor-hover-background: darken( $blue-wordpress, 17% );
+
+	--sidebar-color: $gray-dark;
+	--sidebar-background: lighten( $gray, 30% );
+	--sidebar-gridicon-fill: $gray;
+	--sidebar-heading-color: darken( $gray, 10% );
+	--sidebar-footer-button-color: darken( $gray, 10% );
+	--sidebar-menu-link-secondary-text-color: darken( $gray, 20% );
+	--sidebar-menu-a-first-child-after-background: hex-to-rgb( lighten( $gray, 30% ) );
```

Comment:
`hex-to-rgb()` is a color function that returns the individual RGB values based on a given hexadecimal input color value. This enables the dynamic styling of the overflow backgrounds in the sidebar.